src/pages/database/settings/DBSettings.jsx

```diff
@@ -66,18 +66,18 @@ const Settings = () => {
             <h3>Reload schema</h3>
             <p>Refresh Space Cloud schema, typically required if you have changed the underlying database</p>
             <Button onClick={handleReloadDB}>Reload</Button>
-            <Divider />
+            <div className="divider" style={{ margin: "16px 0px 16px 0px" }} />
```

Comment:
You could have used Divider component only here and given it styles prop. And if you are not using Divider any where in this component, why are you still importing it?